UPGRADE.md

```diff
@@ -0,0 +1,80 @@
+UPGRADE GUIDE FROM 1.x to 2.0
+=============================
+
+This is guide for upgrade from version 1.x to 2.0 for using the CLI tool.
+
+Config file
+-----------
+From now you can create new configuration file: `.php_cs.dist`. This file is prioritized over `.php_cs` configuration file. It is recommended to use first one and add second one to `.gitignore` to allow your contributors to have theirs own configuration file.
```

Comment:
Good point - updated.
If you can enhance it further then please just qoute your proposal.